### DISCUSSION TITLE
New metrics missing hosted source words

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5785] [General] [answered]
The amount of hosted source words is currently missing from the new metrics. Now I need to make some extra calculations to know what the real percentage of source words are actually completed.
I am interested in the actual number of words translated in Polish in one component list (includes all strings without repetition of context). These are the calculations I have to do for this component list:
15 478.21 (total hosted words * 1): 2 (number of all languages) = source words 7 739,105
![1](https://user-images.githubusercontent.com/39633242/113294939-cd11b800-92f7-11eb-9e77-f9a293a4c4f6.jpg)
-------------------------------------------------- -
7 739.105 (source words) - 4754.329 (untranslated words*2) = 2 984.776 (translated words in Polish)
![2](https://user-images.githubusercontent.com/39633242/113295128-13671700-92f8-11eb-9a6c-92a6dc326d06.jpg)
-------------------------------------------------- ---
Now it is enough to calculate what percentage is 2 984.776 from the number 7 739.105 = 38.567%

With two languages it is possible to make such calculations, with more we are no longer able to find out what the real translation progress is (luckily I only have two languages). When translating anything, the most important measure is always the number of source words or characters, we are rarely interested in the number of source strings.

#4981

**Comment by nijel** [comment, marked as answer]
AFAIK it was not present before as well :-).

Anyway I've just extended the table to give more details:

![image](https://user-images.githubusercontent.com/212189/113297960-92aa1a00-92fb-11eb-820d-93d2be6036be.png)

**Reply by tomkolp** [reply]
Each time I am impressed by such a quick time of implementing changes. I also propose to introduce changes in widgets.

**Reply by tomkolp** [reply]
Only further there will be no statistics for individual languages for component lists.
#5792
